components/GroupSingleQ.js

```diff
@@ -1,123 +1,67 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Dimensions, StyleSheet, Text, View } from 'react-native';
-import { Body, Card, CardItem, Input, Item } from 'native-base';
+import { Body, Button, Card, CardItem, Input, Item } from 'native-base';
 import { CheckBox } from 'react-native-elements';
 
 const height = Dimensions.get('window').height;
 
-export default function GroupSingleQ({ question }) {
+export default function GroupSingleQ({ question, answerQuestion }) {
+
+  const [checkWho, setCheckWho] = useState([]);
```

Comment:
Haha, we'll have to revisit the naming conventions, but the increase in functionality today is so helpful!